packages/qwik/src/optimizer/src/plugins/rollup.ts

```diff
@@ -247,11 +247,11 @@ export function normalizeRollupOutputOptionsObject(
 }
 
 export function createRollupError(id: string, diagnostic: Diagnostic) {
-  const loc = diagnostic.highlights[0] ?? {};
+  const loc = diagnostic.highlights?.[0];
```

Comment:
Why are there changes to the optimeter here? This seems unrelated?

Reply:
it broke in a test due to highlights not always being defined. I presumed it was due to the test environment being different, and the highlights are just passed on from swc so we don't control them.